nrf_rpc/include/nrf_rpc.h

```diff
@@ -230,9 +238,17 @@ struct nrf_rpc_err_report {
  *                     a newly allocated packet buffer.
  * @param[in]  _len    Requested length of the packet.
  */
+#ifdef nrf_rpc_tr_alloc_tx_buf
```

Comment:
Maybe it is better to explain this `#ifdef`, because it is not obvious.

```c
/* If nrf_rpc_tr_alloc_tx_buf is a macro, put it directly, because it may allocate memory on stack. */
```